### DISCUSSION TITLE
有计划增加桌面歌词吗，可能比较小众需求

### DISCUSSION THREAD

**Discussion by snfcx** [SPlayer-Dev/SPlayer#167] [想法 Ideas]
RT，我推荐给朋友后他觉得这个软件还缺少一个桌面歌词的功能，其他已经齐全了。
我自己没有使用桌面歌词的习惯也不知道怎么形容。

**Comment by snfcx** [comment]
另外问一下，这个软件快捷键都有那些，没有找到wiki。我想在游戏里面播放暂停都只能退桌面切软件完成。

**Reply by imsyy** [reply]
全局快捷键目前还没有，后续会支持，桌面歌词正在研究怎么搞 👻

**Reply by swfeng00** [reply]
> 全局快捷键目前还没有，后续会支持，桌面歌词正在研究怎么搞 👻
微软商店有个叫热词的第三方软件，功能都很完善，可以考虑和他合作。很多第三方网易云都有和他合作的。你也可以少掉点头发，把精力放在其他事情上。

**Reply by Takenforgranted** [reply]
> 全局快捷键目前还没有，后续会支持，桌面歌词正在研究怎么搞 👻

桌面歌词我记得listen 1好像有，但是桌面歌词的bug很严重，我个人围绕他们的bug issue都写了几个。
